### PR TITLE
The type definition for the property set within _media is insufficient.

### LINKED ISSUE #972: The type definition for the property set within `_media` is insufficient.
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.0

### Description

For example, `aspectRatio` should also accept `minAspectRatio` and `maxAspectRatio`. However, this is not defined.

Add the missing properties using [@media](https://developer.mozilla.org/en-US/docs/Web/CSS/@media#media_features)  as a reference.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by txxxxc**
Closes #972 

## Description

Replace the `_media` property type with strict type.

## Current behavior (updates)

types in `_media` is insufficient(almost all is `StringLiteral`)

## New behavior

types in `_media` is sufficient

## Is this a breaking change (Yes/No):

N/A

## Additional Information

[resolution](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/resolution) property in _media requires a values for example `300dpi`. I could not find out what type is appropriate.

### CODE REVIEW

**Review by 108yen — APPROVED** [review]

File: scripts/generate-css/ui-props.ts
Comment (108yen) [inline scripts/generate-css/ui-props.ts:1]: For literal and number, add `StringLiteral` with or

**Review by 108yen — DISMISSED** [review]

File: scripts/generate-css/ui-props.ts
Comment (txxxxc) [reply scripts/generate-css/ui-props.ts:1]: Thank you for reviewing. fixed in a0a8b6a

File: scripts/generate-css/ui-props.ts [hunk 1]
```diff
@@ -324,34 +324,48 @@ export const uiProps = createUIProps({
...
+      'deviceHeight?: CSS.Property.Height | number | Theme["sizes"]',
+      'minDeviceHeight?: CSS.Property.MinHeight | number | Theme["sizes"]',
+      'maxDeviceHeight?: CSS.Property.MaxHeight | number | Theme["sizes"]',
+      'deviceWidth?: CSS.Property.Width | number | Theme["sizes"]',
+      'minDeviceWidth?: CSS.Property.Width | number | Theme["sizes"]',
+      'mazDeviceWidth?: CSS.Property.Width | number | Theme["sizes"]',
+      'displayMode?: "browser" | "fullscreen" | "minimal-ui" | "picture-in-picture" | "standalone" | "window-controls-overlay" | StringLiteral',
+      'dynamicRange?: "standard" | "high" | StringLiteral',
+      'forcedColors?: "none" | "active" | StringLiteral',
+      "grid?: 0 | 1",
```
Comment (hirotomoyamada) [inline scripts/generate-css/ui-props.ts:349]: This line add `StringLiteral`.

**Comment by hirotomoyamada** [comment]
@txxxxc 

Please run `pnpm gen:css`.

File: scripts/generate-css/ui-props.ts (hunk 1 above)
Comment (txxxxc) [reply scripts/generate-css/ui-props.ts]: fixed in 88bbd6c(Sorry for late).

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**